test/unit/services/util-crypto.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { signatureVerify } from "@polkadot/util-crypto";
+import { waitReady } from '@polkadot/wasm-crypto';
+
+describe('util-crypto', () => {
+
+    beforeEach(async (): Promise<void> => {
+        await waitReady();
+    });
+
+    it('fails with invalid input', () => {
+        const result = signatureVerify(ANOTHER_MESSAGE, HEX_PREFIXED_SIGNATURE, THE_ADDRESS);
+        expect(result.isValid).toBe(false);
+    });
+
+    it('verifies with hex prefixed signature', () => {
+        const result = signatureVerify(THE_MESSAGE, HEX_PREFIXED_SIGNATURE, THE_ADDRESS);
+        expect(result.isValid).toBe(true);
+    });
+});
```

Comment:
Suggestion: move above tests to `signature.service.spec.ts`. By doing so, you do not need that "verifier function" argument anymore.

Reply:
The `VerifyFunction` was introduced to easily mock the method `signatureVerify` from util-crypto. I don't see how moving those tests will affect that.

Reply:
Following your explanation, my suggestion is, indeed, not relevant.